src/apis/Instagram.js

```diff
@@ -0,0 +1,25 @@
+const { APIWrapper } = require('../')
+const fetch = require('node-fetch')
+
+const API_URL = 'https://instagram.com/'
+
+module.exports = class InstagramAPI extends APIWrapper {
+  constructor () {
+    super({
+      name: 'instagram'
+    })
+  }
+
+  getUser (user) {
+    return this.request(user)
+  }
+
+  request (endpoint, queryParams = {}) {
+    const qParams = new URLSearchParams({
+      ...queryParams,
+      __a: 1
+    })
+    return fetch(API_URL + endpoint + `?${qParams.toString()}`)
```

Comment:
axios?

Reply:
why 😭

Reply:
- Axios performs automatic transformations of JSON data.
- Axios is more secure, featuring built in cross site forgery (XSRF) protection
- Axios has better error handling
- Axios has interceptors
- Axios has the ability to monitor POST request progress

Reply:
- said @fernando457829